controller.go

```diff
@@ -480,3 +497,63 @@ func (c *Controller) removeMember(memberDID string) (map[string]string, error) {
 	}
 	return map[string]string{"status": "ok"}, nil
 }
+
+func (c *Controller) startBitcoind() (*BitcoindCtlResponse, error) {
+	req, err := http.NewRequest("POST", viper.GetString("bitcoind_ctl.endpoint")+"/start", nil)
+	if err != nil {
+		log.WithError(err).Error("fail to create bitcoind-ctl api request")
+		return nil, fmt.Errorf("fail to create bitcoind-ctl api request")
+	}
+	result, err := c.doHttpRequest(req)
+	if err != nil {
+		log.WithError(err).Error("fail to call bitcoind-ctl api")
+		return nil, fmt.Errorf("fail to call bitcoind-ctl api")
+	}
+	return result, nil
+}
+
+func (c *Controller) stopBitcoind() (*BitcoindCtlResponse, error) {
+	req, err := http.NewRequest("POST", viper.GetString("bitcoind_ctl.endpoint")+"/stop", nil)
+	if err != nil {
+		log.WithError(err).Error("fail to create bitcoind-ctl api request")
+		return nil, fmt.Errorf("fail to create bitcoind-ctl api request")
+	}
+	result, err := c.doHttpRequest(req)
+	if err != nil {
+		log.WithError(err).Error("fail to call bitcoind-ctl api")
+		return nil, fmt.Errorf("fail to call bitcoind-ctl api")
+	}
+	return result, nil
+}
+
+func (c *Controller) getBitcoindStatus() (*BitcoindCtlResponse, error) {
+	req, err := http.NewRequest("GET", viper.GetString("bitcoind_ctl.endpoint")+"/status", nil)
+	if err != nil {
+		log.WithError(err).Error("fail to create bitcoind-ctl api request")
+		return nil, fmt.Errorf("fail to create bitcoind-ctl api request")
+	}
+	result, err := c.doHttpRequest(req)
+	if err != nil {
+		log.WithError(err).Error("fail to call bitcoind-ctl api")
+		return nil, fmt.Errorf("fail to call bitcoind-ctl api")
+	}
+	return result, nil
+}
+
+func (c *Controller) doHttpRequest(req *http.Request) (*BitcoindCtlResponse, error) {
```

Comment:
This `doHttpRequest` only works well with `bitcoind-ctl` commands. A more specific name could be used or write a dedicated client for the interactions with bitcoind-ctl.